backend/ENVIRONMENT.md

```diff
@@ -5,6 +5,7 @@ This application recognizes the following environment variables:
 | Name                   | Description                                                                                                                          |          Range/Type           | Required? | Default |
 | ---------------------- | ------------------------------------------------------------------------------------------------------------------------------------ | :---------------------------: | :-------: | :-----: |
 | `API_PORT`             | HTTP port that the application listens on                                                                                            |         1025 - 65535          |           |  3000   |
+| `WORKERAPI_PORT`       | HTTP port that the application listens on                                                                                            |         1025 - 65535          |           |  3000   |
```

Comment:
```suggestion
```

Not needed--the current setup has all services (including workers) take the default `3000` for the internal port

Reply:
Also, this env is for the SAT backend itself, not the Gateway services